soap/src/java/com/zimbra/soap/JaxbUtil.java

```diff
@@ -1093,7 +1093,55 @@
             com.zimbra.soap.voice.message.UploadVoiceMailRequest.class,
             com.zimbra.soap.voice.message.UploadVoiceMailResponse.class,
             com.zimbra.soap.voice.message.VoiceMsgActionRequest.class,
-            com.zimbra.soap.voice.message.VoiceMsgActionResponse.class
+            com.zimbra.soap.voice.message.VoiceMsgActionResponse.class,
+
+            //zimbra mobile gateway
+            com.zimbra.soap.account.message.BootstrapMobileGatewayAppRequest.class,
+            com.zimbra.soap.account.message.BootstrapMobileGatewayAppResponse.class,
+            com.zimbra.soap.account.message.RenewMobileGatewayAppTokenRequest.class,
+            com.zimbra.soap.account.message.RenewMobileGatewayAppTokenResponse.class,
+            com.zimbra.soap.account.message.RegisterMobileGatewayAppRequest.class,
+            com.zimbra.soap.account.message.RegisterMobileGatewayAppResponse.class,
+            com.zimbra.soap.account.message.GetGcmSenderIdRequest.class,
+            com.zimbra.soap.account.message.GetGcmSenderIdResponse.class,
+
+            //Oauth
+            com.zimbra.soap.account.message.GetOAuthConsumersRequest.class,
+            com.zimbra.soap.account.message.GetOAuthConsumersResponse.class,
+            com.zimbra.soap.account.message.RevokeOAuthConsumerRequest.class,
+            com.zimbra.soap.account.message.RevokeOAuthConsumerResponse.class,
+
+            //smime
+            com.zimbra.soap.mail.message.SendSecureMsgRequest.class,
+            com.zimbra.soap.mail.message.SendSecureMsgResponse.class,
+            com.zimbra.soap.account.message.GetSmimeCertificateInfoRequest.class,
+            com.zimbra.soap.account.message.GetSmimeCertificateInfoResponse.class,
+            com.zimbra.soap.account.message.SaveSmimeCertificateRequest.class,
+            com.zimbra.soap.account.message.SaveSmimeCertificateResponse.class,
+
+            //IMAP
+            com.zimbra.soap.mail.message.ListIMAPSubscriptionsRequest.class,
```

Comment:
I recently sorted this list and removed duplicates.  I notice that you have duplicates in here - please remove them.

Reply:
Hm not sure what happened here, maybe merge error?